templates/consul.systemd.j2

```diff
@@ -8,7 +8,8 @@ Environment="GOMAXPROCS=`nproc`"
 Restart=on-failure
 User=consul
 Group=consul
-ExecStart=/usr/local/bin/consul agent -config-dir /etc/consul
+ExecStart=/usr/local/bin/consul agent -config-dir /etc/consul{% if consul_ui_enabled|default(false)|bool %} -ui{% endif %}
```

Comment:
I would prefer to have it in the config file https://www.consul.io/docs/agent/options.html#ui